nep-1.mediawiki

```diff
@@ -33,13 +33,13 @@ The NEP process begins with a new idea for NEO. It is highly recommended that a
 
 Each NEP must have a champion - someone who writes the NEP using the style and format described below, shepherds the discussions in the appropriate forums, and attempts to build community consensus around the idea.
 
-Vetting an idea publicly before going as far as writing an NEP is meant to save the potential author time. Asking the NEO community first if an idea is original helps prevent too much time being spent on something that is guaranteed to be rejected based on prior discussions (searching the Internet does not always do the trick). It also helps to make sure the idea is applicable to the entire community and not just the author. Just because an idea sounds good to the author does not mean it will work for most people in most areas where NEO is used. Examples of appropriate public forums to gauge interest around your NEP include [https://www.reddit.com/r/NEO the NEO subreddit], [https://github.com/neo-project/proposals/issues the Issues section of this repository], and [http://slack.neo.org one of the NEO Slack channels]. In particular, [https://github.com/neo-project/proposals/issues the Issues section of this repository] is an excellent place to discuss your proposal with the community and start creating more formalized language around your NEP.
+Vetting an idea publicly before going as far as writing an NEP is meant to save the potential author time. Asking the NEO community first if an idea is original helps prevent too much time being spent on something that is guaranteed to be rejected based on prior discussions (searching the Internet does not always do the trick). It also helps to make sure the idea is applicable to the entire community and not just the author. Just because an idea sounds good to the author does not mean it will work for most people in most areas where NEO is used. Examples of appropriate public forums to gauge interest around your NEP include [https://www.reddit.com/r/NEO the NEO subreddit], [https://github.com/neo-project/proposals/issues the Issues section of this repository], and [https://discord.io/neo the NEO Discord]. In particular, [https://github.com/neo-project/proposals/issues the Issues section of this repository] is an excellent place to discuss your proposal with the community and start creating more formalized language around your NEP.
```

Comment:
@corollari, this part is looks wrong:

" [https://github.com/neo-project/proposals/issues the Issues section of this repository]"
Perhaps the link needs a  ] before

Reply:
I just double checked and it should be alright, maybe the confusion stems from the fact that these are [mediawiki-style](https://www.mediawiki.org/wiki/Help:Links#External_links) links instead of markdown-style ones?